### PR TITLE
BXMSPROD-702 jbpm-installer excluded from productized profile

### PR DESCRIPTION
**Pull request by Ginxo**
https://issues.redhat.com/browse/BXMSPROD-702
My only concern is about the jbpm-installer sources remain in the jbpm-distribution assembly zip `jbpm-%version%-src-zip` file

### CODE REVIEW

**Comment by mareknovotny** [comment]
I tried that it seems works only with property activation,if you use `-P productized` it is still included in build reactor

**Comment by mareknovotny** [comment]
> https://issues.redhat.com/browse/BXMSPROD-702
> My only concern is about the jbpm-installer sources remain in the jbpm-distribution assembly zip `jbpm-%version%-src-zip` file
we need to exclude it there too

File: pom.xml [hunk 1]
```diff
@@ -279,6 +278,29 @@
...
+                <property>
+                    <name>productized</name>
+                    <value>false</value>
+                </property>
+            </activation>
+            <modules>
+                <module>jbpm-installer</module>
+            </modules>
+        </profile>
     </profiles>
```
Comment (mareknovotny) [inline pom.xml:304]: I think both cases/conditions can be covered by simple 
`<value>!true</value>`

Comment (Ginxo) [reply pom.xml:304]: let me try it

Comment (Ginxo) [reply pom.xml:293]: @mareknovotny much simpler with
```
<name>productized</name>
<value>!true</value>
```
thanks a lot

**Comment by Ginxo** [comment]
@mareknovotny also excluded from src distribution zip file

File: pom.xml [hunk 2]
```diff
@@ -279,6 +278,18 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>jbpm-installer-exclude-productized</id>
```
Comment (afalhambra) [inline pom.xml:282]: Why not using same id profile as per jbpm-distribution/pom.xml in line 404 
```suggestion
            <id>jbpm-installer-include</id>
```

Comment (afalhambra) [inline pom.xml:282]: Why not using same id profile as per jbpm-distribution/pom.xml in line 404 
```suggestion
            <id>jbpm-installer-include</id>
```

**Comment by mareknovotny** [comment]
There could be another idea how to do that in more simpler way https://github.com/kiegroup/jbpm/pull/1631, I am not sure that we need full profile there, but kept it for not changing the usages
Also jbpm-distribution could be needed somewhere in assemblies, but we can look if that is safe

**Comment by Ginxo** [comment]
replaced by https://github.com/kiegroup/jbpm/pull/1631

**Pull request closed without merging**